<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameLineEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>190</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>240</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>240</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>190</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="errorLabel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="setBudgetButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>240</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Budget</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Welcome to WealthWay</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hintButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>280</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>HINT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mainManagerPage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>120</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="descriptionLineEdit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>167</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="amountLineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>87</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="addExpenseButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>90</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Expense</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="expenseTable">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>551</width>
        <height>271</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>135</number>
      </attribute>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Category</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>87</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Amount:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>127</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Category:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>167</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Description: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>207</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeExpenseButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>140</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Expense</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewExpenseButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>190</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>View Expenses</string>
      </property>
     </widget>
     <widget class="QLabel" name="budgetDisplayLabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="categoryComboBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Food</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Entertainment</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Transportation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rent</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Other</string>
       </property>
      </item>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>207</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>WealthWay</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="viewExpensePage">
     <widget class="QPushButton" name="backToManagerButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>460</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="expenseViewTable">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>531</width>
        <height>401</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>130</number>
      </attribute>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Category</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="minPriceFilter">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="maxPriceFilter">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>300</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="categoryComboBoxView">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Food</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Entertainment</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Transportation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rent</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Other</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="filterPriceButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter Price</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter Category: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter Minimum Price: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter Maximum Price: </string>
      </property>
     </widget>
     <widget class="QLabel" name="budgetDisplayLabel_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="totalBudgetUsedLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="overBudgetLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>511</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="remainingBudgetLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>451</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>WealthWay</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generateSummaryReportButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>30</y>
        <width>181</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Summary Report</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generateGraphicalReportButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>70</y>
        <width>181</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Graphical Report</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
